README.md

```diff
@@ -1,70 +1,204 @@
+
+-   [Juicebox Docs](https://docs.juicebox.money/dev/)
+-   [Juicebox Github](https://github.com/jbx-protocol)
+-   Goerli Etherscan Link Placeholder for the deployed [DominantJuice.sol]()
+
+<!-- DOMINANT ASSURANCE BACKGROUND -->
+
+## Dominant Assurance Background
+
+Only [23.6%](https://www.thecrowdfundingcenter.com/data/projects) of crowdfunding campaigns succeed, some of which is due to the [“free rider” problem](https://www.investopedia.com/terms/f/free_rider_problem.asp). History has also shown that campaigns that have [reached at least 30%](https://www.fundera.com/resources/crowdfunding-statistics) of their funding in the first week have a greater chance of achieving their final goal. [Dominant assurance](https://foresight.org/summary/dominant-assurance-contracts-alex-tabarrok-george-mason-university/) seeks to simultaneously minimize free riders while ensuring earlier pledging.
+
+The key concept lies in incentivizing early and significant contributions. Before the start of a crowdfunding campaign, the campaign creator locks their own funds in a dominant assurance smart contract that they or their team owns. If the campaign fails, pledgers get a full refund along with a portion of the locked funds, as a refund bonus, according to a custom formula that is based on how early and large their pledge was. Ideally, the refund bonus formula is a type of exponential decay. The secret sauce is in finding the right formula for your project that incentivizes pledgers to raise funds above the 30% threshold very early in the campaign. If the campaign succeeds, the pledgers get the goods but no bonuses, and the complete refund bonus can be wihdrawn by the campaign owner after the campaign ends. This creates a win-win situation for early pledgers, who either get a return on their pledge or get the desired good if the campaign goal is met.
+
+Overall, the transparency and deterministic characteristics of smart contracts help to minimize pledger risk and encourage wider participation, increasing chances of success. Using the dominant assurance mechanism also sends a message to your potential pledgers that you believe in your project so much that you are willing to put your own money at stake for the success of the project.
+
+<!-- CAMPAIGN WORKFLOW -->
+
+## Campaign Workflow
+
+### Campaign Setup
+
+-   The project team executes the LaunchProjectAndDeployDAC.s.sol script by running `forge script` with the specific project parameters (owner address, cycle target, cycle start time, duration, minimum pledge amount, and project metadata).
```

Comment:
does the forge script runner have a --help or something? if not, some examples of running the scrips would be helpful

Reply:
Ahhh, man, I knew I was forgetting something. Yes, I will get this done. And I believe there is a help feature, but I was going to put full examples as well, which will be super needed in the README. It took a while to piece all the scripting commands together. As far as I can tell, there are only like three projects on Github that show full examples with user-provided inputs, so will be great to provide a small contribution towards that.